src/lib.rs

```diff
@@ -154,7 +154,7 @@ pub enum DeepgramError {
 }
 
 #[cfg_attr(not(feature = "listen"), allow(unused))]
-type Result<T> = std::result::Result<T, DeepgramError>;
+type Result<T, E = DeepgramError> = std::result::Result<T, E>;
```

Comment:
Making the error type optional means you can `use deepgram::Result;`, but still specify other error types where you need them.